### ISSUE TITLE
[2.1.1-47-beta] Question re update trigger functionality

### ISSUE THREAD

**Issue by AnnaSchrimpf** [SanaMobile/vaccine_registry#96]
Could you clarify whether the "check now" button via client UI is the only way to trigger update? My understanding is that we would also be able to "force" an update without ANM doing triggering the update - is this correct?

**Comment by kepaik** [comment]
The "Check now" button is only the application update.
Do you think it would be helpful to add a "Sync Now" button that would force a complete update like a first install?

**Comment by AnnaSchrimpf** [comment]
Sorry if this was imprecise, I am talking about the update of the client, not syncing of records. 

My understanding is that records are synced continuously if internet connection permits, is this incorrect?

**Comment by kepaik** [comment]
The app does check the server intermittently to check the latest version and will download it automatically. However, it will be up to the user to "accept" the update and install when the download is complete.

Correct about the data sync. The green bar actually flashes between states that are not perceptible when there is a decent connection.

**Comment by AnnaSchrimpf** [comment]
Ok, I see. In that case, it would be useful if we could produce screenshots for the update process (I know it's not complicated but the ANMs will be taken by surprise/panic if we don't prepare them for this scenario).

**Comment by kepaik** [comment]
Would you rather it just be on demand rather than automatic?

**Comment by AnnaSchrimpf** [comment]
No, I think having the option to do this automatically is preferable. Otherwise we need to rely on ANMs to check regularly, which I don't think we can.
